Options.py

```diff
@@ -96,7 +101,7 @@ def meta__init__(self, *args, **kwargs):
 
 class Option(typing.Generic[T], metaclass=AssembleOptions):
     value: T
-    default = 0
+    default: typing.ClassVar[typing.Any]  # something that __init__ will be able to convert to the correct type
```

Comment:
It feels counterintuitive to a different type for the default than `T`, but since most default values are immutable whereas `T` is not, I think it makes sense.